components/patient-tools/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1 @@
+Phenotips.FormSection.suggestTermPlaceholder=enter free text and choose among suggested ontology terms
```

Comment:
While we're refactoring this, can you also replace `ontology` with `vocabulary please? Same for the Spanish translation below.